docs/03-pages/02-api-reference/01-components/head.mdx

```diff
@@ -57,7 +59,15 @@ In this case only the second `<meta property="og:title" />` is rendered. `meta`
 
 > The contents of `head` get cleared upon unmounting the component, so make sure each page completely defines what it needs in `head`, without making assumptions about what other pages added.
 
+## Use minimal nesting
+
 `title`, `meta` or any other elements (e.g. `script`) need to be contained as **direct** children of the `Head` element,
 or wrapped into maximum one level of `<React.Fragment>` or arrays—otherwise the tags won't be correctly picked up on client-side navigations.
 
-> We recommend using [next/script](/docs/pages/building-your-application/optimizing/scripts) in your component instead of manually creating a `<script>` in `next/head`.
+## Use `next/script` for scripts
+
+We recommend using [`next/script`](/docs/pages/building-your-application/optimizing/scripts) in your component instead of manually creating a `<script>` in `next/head`.
+
+## No `html` or `body` tags
+
+You **cannot** use `<Head>` to set attributes on `<html>` or `<body>` tags. Doing so will result in an `next-head-count is missing` error. `next/head` can only handle tags inside the HTML `<head>` tag. [Learn more](https://nextjs.org/docs/pages/api-reference/components/head).
```

Comment:
```suggestion
You **cannot** use `<Head>` to set attributes on `<html>` or `<body>` tags. This will result in an `next-head-count is missing` error. `next/head` can only handle tags inside the HTML `<head>` tag.
```